packages/components/col/__tests__/layout.spec.tsx

```diff
@@ -85,16 +73,19 @@ describe('Col', () => {
   })
 
   it('responsive', () => {
-    const TestComponent = {
-      template: `<el-row :gutter="20">
-      <el-col ref="col" :sm="{ span: 4, offset: 2 }" :md="8" :lg="{ span: 6, offset: 3 }">
-      </el-col>
-    </el-row>`,
-      components: {
-        'el-col': Col,
-        'el-row': Row,
-      },
-    }
+    const TestComponent = defineComponent({
+      setup: () => () =>
+        (
+          <Row gutter={20}>
+            <Col
+              ref="col"
+              sm={{ span: 4, offset: 2 }}
+              md={8}
+              lg={{ span: 6, offset: 3 }}
+            />
+          </Row>
+        ),
+    })
     const wrapper = mount(TestComponent)
```

Comment:
Can we do:

```tsx
mount( <Row gutter={20}>
            <Col
              ref="col"
              sm={{ span: 4, offset: 2 }}
              md={8}
              lg={{ span: 6, offset: 3 }}
            />
          </Row>)
```
instead of creating a component?

Reply:
We can't. The unit test will be failed.